lib/mayaUsd/undo/OpUndoItems.cpp

```diff
@@ -15,6 +15,8 @@
 //
 #include "OpUndoItems.h"
 
+#include "OpUndoItemMuting.h"
```

Comment:
According to our coding standard only the associated header file should be included with "
All other public headers should use <

Reply:
Ah, OK, I thought it was all the ones in the same folder. I'll fix that.